gitpod-io/website
# Update instructions for "Trying out changes to your Dockerfile in the existing workspace"

## Description

Update instructions for "Trying out changes to your Dockerfile in the existing workspace"

## Related Issue(s)

- https://github.com/gitpod-io/gitpod/pull/15638.

## How to test

Open the preview environment of this PR and check the section "Trying out changes to your Dockerfile in the existing workspace": https://deploy-preview-3226--gitpod-kumquat.netlify.app/docs/configure/workspaces/workspace-image#in-the-existing-workspace

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3226"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
